README.md

```diff
@@ -1,3 +1,9 @@
+# Opendoor-related CHANGELOG
+
+* 1.5.4-opendoor.1
```

Comment:
I still prefer SEMVER versions

helm charts and its ecosystem (artifactory, CLI tools, etc) expects SEMVER version, and this may break things (e.g. helm chart publish to ECR has a restriction on SEMVER version)